x-pack/test/screenshot_creation/apps/response_ops_docs/stack_alerting/list_view.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const commonScreenshots = getService('commonScreenshots');
+  const screenshotDirectories = ['response_ops_docs', 'stack_alerting'];
+  const pageObjects = getPageObjects(['common', 'header']);
+  const rules = getService('rules');
+  const supertest = getService('supertest');
+
+  describe.only('list view', function () {
```

Comment:
The `.only` is useful for local debugging, but should not be committed.

Reply:
oops!